apps/web/package.json

```diff
@@ -6,17 +6,26 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "eslint --fix './'",
+    "clean": "rm -rf .turbo && rm -rf node_modules"
```

Comment:
When do we need to run this? @kilemensi

Reply:
That's part of the review @gertie-sheshe. When do you **_think_** we'd need this (if at all)?

Reply:
Part of the review is to ask questions where we do not understand, not to explain 😭😭😭 Wawu @kilemensi

Reply:
Welcome to the **new school** @gertie-sheshe .... basically whenever you need to reset the repo e.g. you're having issues with turbo cache, call `pnpm clean`, having issues with dependencies and you'd like to run `pnpn install` from scratch, `call pnpm clean`, etc., etc.